packages/sdks-tests/src/e2e-tests/personalization-container.spec.ts

```diff
@@ -182,6 +182,8 @@ test.describe('Personalization Container', () => {
   test('only default variants are ssred on the server', async ({ browser, packageName, sdk }) => {
     test.skip(!isSSRFramework(packageName));
     test.skip(!['react', 'oldReact'].includes(sdk));
+    // Cannot read properties of null (reading 'useContext')
```

Comment:
@samijaber we should create a ticket to move away from remix v1 for gen1 tests and use v2 that we already have in snippets. This will help us alias a single react and fix these tests

repo: https://github.com/sidmohanty11/builder-remix-gen1-example